src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,4 @@
+$text-color-grey: #616070;
+$text-color-black: #060B35;
+$color-white: #fff;
+$color-blue: #00ACDC;
```

Comment:
Variable names should describe their purpose. As if the designer changes the color to red instead, it will be strange when `$color-blue` will be red. For example `$main-color`